Formula/rustscan.rb

```diff
@@ -0,0 +1,19 @@
+class Rustscan < Formula
+  desc "Modern Day Portscanner"
+  homepage "https://github.com/rustscan/rustscan"
+  url "https://github.com/RustScan/RustScan/archive/1.9.0.tar.gz"
+  sha256 "9923b91310be342f5fcec42796662b53f4d0b05ec5339c44b801959ebfe824dc"
+  license "GPL-3.0-or-later"
+
+  depends_on "rust" => :build
+  depends_on "nmap"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/rustscan", "--greppable", "127.0.0.1"
+    system "#{bin}/rustscan", "--greppable", "0.0.0.0"
```

Comment:
Can this be checked for output? Something like this
```suggestion
    assert_match "error", shell_output("#{bin}/rustscan --greppable 127.0.0.1")
    assert_match "error", shell_output("#{bin}/rustscan --greppable 0.0.0.0")
```

Reply:
Hey! This assumes that RustScan, when it errors, prints only the word "error".

I'm trying to figure out how to do `assert_no_match` and check to see if the result contains the world "panic" (which is an error that is guaranteed to be in all RustScan error messages), but my Ruby skills are lackluster :sweat_smile: